substrate/client/informant/src/lib.rs

```diff
@@ -52,13 +51,14 @@ where
 		.filter_map(|_| async {
 			let net_status = network.status().await;
 			let sync_status = syncing.status().await;
+			let num_connected_peers = syncing.num_connected_peers();
```

Comment:
`SyncingStrategy::num_peers()` (or equivalent) should be used instead, because `SyncingEngine::num_connected` also counts light peers, and we don't report the light peers to the informant.

Reply:
Actually we do, that is exactly the problem that https://github.com/paritytech/substrate/pull/13700 addressed and why `num_connected_peers` was added to `SyncStatus`. Before this PR `num_connected_peers` was patched in here:
https://github.com/paritytech/polkadot-sdk/blob/b3c2a25b73bb4854f26204068f0aec3e8577196c/substrate/client/network/sync/src/engine.rs#L763-L767

I just made it accessible in a different way, but the underlying logic is the same.

Reply:
Yeah, you are right. Sorry :see_no_evil: